### ISSUE TITLE
WCAG 4.1.2 test 16: false positive?

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#88]
[This file](https://github.com/veraPDF/veraPDF-wcag-algs/files/7374561/tabel-zonder-lege-alinea.pdf) fails on **WCAG 4.1.2 test 16** ('_Paragraph structure element is empty_'), but it does not appear to contain empty paragraphs.

What probably causes this error is that the table cells contain paragraphs with artefacts. This is probably why the file **also fails PDF/UA 7.2 test 10** ('_TR (table row) may contain only table cells (TH or TD elements)_')

In [this other file](https://github.com/veraPDF/veraPDF-wcag-algs/files/7374622/tabel-zonder-lege-alinea-Ppad-verwijderd.pdf) were the P have been removed, the two errors do not occur.

What do you think is the correct error to show when you come accross this sort of situation where there are paragraphs with artefacts in a table?

**Comment by jzuidweg** [comment]
Ok, I understand that WCAG 4.1.2 test 16 does not give a false positive here. This issue can be closed.
